pipelines/release-images-cf-deployment/tasks/build_release.sh

```diff
@@ -41,8 +45,12 @@ function build_release() {
   built_image_tag=$(echo "${built_image_filter}" | awk '{ printf $2 }')
   built_image="${built_image_repository}:${built_image_tag}"
   echo -e "Built image: ${GREEN}${built_image}${NC}"
-  if curl --head --fail "https://hub.docker.com/v2/repositories/${built_image_repository}/tags/${built_image_tag}/" 1> /dev/null 2> /dev/null; then
-    echo "Skipping push for ${GREEN}${built_image}${NC} as it is already present in the registry..."
+  docker_creds_string=""
+  if [ ! -z "${docker_username}" ] && [ ! -z "${docker_password}" ]; then
+      docker_creds_string="-u ${docker_username}:${docker_password}"
+  fi
+  if curl --silent ${docker_creds_string} https://${docker_registry}/v2/${docker_organization}/${built_image_repository}/manifests/${built_image_tag} | jq '.errors[0].code' | grep -q null; then
```

Comment:
same here, double quote:
- ${docker_creds_string}
- ${docker_registry}
- ${docker_organization}
- ${built_image_repository}
- ${built_image_tag}

you can see this requested changes reasoning in https://github.com/koalaman/shellcheck/wiki/SC2086

Reply:
DONE!